src/Agent/NewRelic/Agent/Extensions/NewRelic.Agent.Extensions/Collections/ConcurrentHashSet.cs

```diff
@@ -47,7 +47,13 @@ public void Add(T item)
                 _hashSet.Add(item);
             }
         }
-
+        public bool TryAdd(T item)
```

Comment:
With the addition of `TryAdd()` do we really need `Add()`?

Reply:
Yep, `ICollection` requires an `Add` that returns void.